doc/install-on-docker.md

```diff
@@ -23,21 +66,45 @@ cd ray
 
 This script creates several Docker images:
 
- * The `ray-project/ray:deploy` image is a self-contained copy of code and binaries suitable for end users.
- * The `ray-project/ray:examples` adds additional libraries for running examples.
- * Ray developers who want to edit locally on the host filesystem should use the `ray-project/ray:devel` image, which allows local changes to be reflected immediately within the container.
+ * The `ray-project/deploy` image is a self-contained copy of code and binaries suitable for end users.
+ * The `ray-project/examples` adds additional libraries for running examples.
+ * The `ray-project/base-deps` image builds from Ubuntu Xenial and includes Anaconda and other basic dependencies and can serve as a starting point for developers.
+
+Review images by listing them:
+```
+$ docker images
+```
+
+Output should look something like the following:
+```
+REPOSITORY                          TAG                 IMAGE ID            CREATED             SIZE
+ray-project/examples                latest              7584bde65894        4 days ago          3.257 GB
+ray-project/deploy                  latest              970966166c71        4 days ago          2.899 GB
+ray-project/base-deps               latest              f45d66963151        4 days ago          2.649 GB
+ubuntu                              xenial              f49eec89601e        3 weeks ago         129.5 MB
+```
+
 
 ## Launch Ray in Docker
 
 Start out by launching the deployment container.
 
 ```
-docker run --shm-size=1024m -t -i ray-project/ray:deploy
+docker run --shm-size=<shm-size> -t -i ray-project/ray:deploy
```

Comment:
This command doesn't work, I think you need to remove the `ray:` and just use `ray-project/deploy`